src/test/java/com/stripe/model/PagingIteratorTest.java

```diff
@@ -168,4 +236,28 @@ void testAutoPaginationWithoutApiKey() throws StripeException {
           collection.autoPagingIterable();
         });
   }
+
+  @Test
+  void testAutoPaginationRequestOptionsPropagation() throws StripeException {
+    // set some arbitrary parameters so that we can verify that they're
+    // used for requests on ALL pages
+    final Map<String, Object> page0Params = new HashMap<>();
+    page0Params.put("foo", "bar");
```

Comment:
Assert that the param is sent with all pages?

Reply:
The comment was copy pasta, that doesn't make sense with this test case imo.